backend/server.js

```diff
@@ -50,7 +50,12 @@ app.get("/api/allColleges", (req, res) => {
 
 // editCollege
 // edits an existing college entry with passed in params
-
+app.get("/api/editCollege", (req, res) => {
```

Comment:
was looking at this PR to see how to structure my own lol, but just noticed this route should prob be a PUT or a POST request since it's mutating data :)

Reply:
oop yes that is wrong,, thanks!